index.html

```diff
@@ -3104,6 +3110,288 @@ <h3>Using the Document Base for the Default Vocabulary</h3>
+         data-result-for="A sealed @context with an exception-original">
+    <!--
+    [{
+      "http://schema.org/name": [{"@value": "Digital Bazaar"}],
+      "http://schema.org/member": [
+        {
+          "@type": ["http://xmlns.com/foaf/0.1/Person"],
+          "http://schema.org/name": [{"@value": "Manu Sporny"}]
+        }
+      ]
+    }]
+    -->
+    </pre>
+    <table class="statements"
+           data-result-for="A sealed @context with an exception-expanded"
+           data-to-rdf>
+      <thead><tr><th>Subject</th><th>Property</th><th>Value</th></tr></thead>
+      <tbody>
+        <tr><td>_:b0</td><td>schema:name</td><td>Digital Bazaar</td></tr>
+        <tr><td>_:b0</td><td>schema:member</td><td>_:b1</td></tr>
+        <tr><td>_:b1</td><td>rdf:type</td><td>foaf:Person</td></tr>
+        <tr><td>_:b1</td><td>schema:name</td><td>Manu Sporny</td></tr>
+      </tbody>
+    </table>
+    <pre class="turtle"
+         data-content-type="text/turtle"
+         data-result-for="A sealed @context with an exception-expanded"
+         data-transform="updateExample"
+         data-to-rdf>
+      <!--
+      @prefix foaf: <http://xmlns.com/foaf/0.1/> .
+      @prefix schema: <http://schema.org/> .
+
+      [
+        schema:name "Digital Bazaar";
+        schema:member [
+          a foaf:Person;
+          schema:name "Manu Sporny"
+        ]
+      ] .
+      -->
+    </pre>
+  </aside>
+
+  <p>While sealed term definitions can not be directly overridden,
+      it is worth noting that setting <code>@context</code> to <code>null</code>
+      will erase everything from the active context,
+      <em>including</em> sealed term definitions.
+      </p>
+
+  <aside class="example ds-selector-tabs changed"
+    title="@context null erases sealed term definitions">
+    <div class="selectors">
+        <button class="selected" data-selects="original">Original</button>
+        <button data-selects="expanded">Expanded</button>
+        <button data-selects="statements">Statements</button>
+        <button data-selects="turtle">Turtle</button>
+        <a class="playground" target="_blank"></a>
+    </div>
+    <pre class="original selected nohighlight" data-transform="updateExample">
+      <!--
+      {
+        "@context": [
+          {
+            ****"@version": 1.1****,
+            ****"@sealed": true****,
+            "Organization": "http://schema.org/Organization",
+            "name": "http://schema.org/name",
+            "employee": {
+              "@id": "http://schema.org/employee",
+              ****"@context": null****
+            }
+          }
+        ],
+        "@type": "Organization",
+        "name": "Digital Bazaar",
+        "employee" : {
+          #### -- because of "@context": null in the scoped context, ####
+          #### -- the active context at this point is empty; ####
+          #### -- so we can (and we must) redefine "name" below ####
+          "@context": {
```

Comment:
Note that `--` inside of `<!-- -->` is causing the HTML parser problems. Try using a unicode character such add `–` instead.